web/src/features/charts/elements/AreaGraphLayers.tsx

```diff
@@ -100,12 +110,26 @@ function AreaGraphLayers({
           ]
         );
 
+        const isCurrentLayerSelected =
+          showHoverHighlight && hasSelectedLayer && selectedLayerIndex === ind;
+        const shouldLayerBeSaturated =
+          isSingleLayer || !hasSelectedLayer || isCurrentLayerSelected;
+
+        const emphasizeStroke = !isSingleLayer && isCurrentLayerSelected;
+        let stroke = layer.stroke;
+        if (showHoverHighlight && emphasizeStroke) {
+          stroke = isDarkModeEnabled ? 'white' : 'black';
+        }
+
         return (
           <React.Fragment key={layer.key}>
             <path
-              className={layers.length > 1 ? 'sm:hover:opacity-75' : ''}
+              className={
+                shouldLayerBeSaturated ? 'sm:hover:opacity-100' : 'sm:opacity-30'
```

Comment:
```suggestion
                shouldLayerBeSaturated ? 'sm:hover:opacity-100' : 'sm:opacity-50'
```

Maybe personal preference but this feels a bit too much on dark more. I think 50 is enough now that we have the stroke outline 🙂

Reply:
I think that's reasonable. I'll run it by @Alportan too so we're all on the same page

Reply:
I think I'll go ahead and merge this in as-is. It's pretty easy to update if we want to change it after